chips/src/main/java/com/decathlon/vitamin/compose/chips/VitaminChipButtons.kt

```diff
@@ -0,0 +1,32 @@
+package com.decathlon.vitamin.compose.chips
+
+import androidx.compose.material.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+
+object VitaminChipButtons {
+    @Composable
+    fun Close(
+        modifier: Modifier = Modifier,
+        contentDescription: String? = null,
+    ) {
+        Icon(
+            modifier = modifier,
+            painter = painterResource(R.drawable.ic_vtmn_close_line),
+            contentDescription = contentDescription
+        )
+    }
+
+    @Composable
+    fun Selected(
```

Comment:
It is the name from Figma?

At the first read (check my comment on your `README.md` file), I didn't understand why this component was the default value for one variant of our `Chip` but now I understand.

Maybe, `Check` is a better name? What do you think?

Reply:
Yes, you're right, `Check` is more logical